Description: Always install systemd service
 Checking if libsystemd-dev is installed is not a good proxy to decide to
 install a systemd service. Moreover, it does not work despite the pkgconfig
 provided in libsystemd-dev. Always install the systemd service.
Author: Agathe Porte <agathe.porte@oss.qualcomm.com>
Last-Update: 2026-07-15

Index: pkg-urm/CMakeLists.txt
===================================================================
--- pkg-urm.orig/CMakeLists.txt
+++ pkg-urm/CMakeLists.txt
@@ -67,7 +67,4 @@ configure_file(
 @ONLY)
 
 # Install service
-pkg_check_modules(PC_SYSTEMD QUIET systemd)
-if(PC_SYSTEMD_FOUND)
   install(FILES ${CMAKE_CURRENT_BINARY_DIR}/urm.service DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/systemd/system/)
-endif()
